### PR TITLE
.*: Add DCO and CONTRIBUTING.md files

### PR DESCRIPTION
**Pull request by ajnavarro**
Signed-off-by: Antonio Jesus Navarro Perez <antnavper@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# source{d} Contributing Guidelines
```
Comment (mcuadros) [inline CONTRIBUTING.md]: The contributing guide should explain how to setup the development environment.

Comment (ajnavarro) [reply CONTRIBUTING.md]: Shall we add it to the CONTRIBUTING.md template? https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md

Comment (smola) [reply CONTRIBUTING.md]: Maybe we should just add a `Tests` section to README? So that there is: Installation, Usage, Build, Tests.

**Comment by smola** [comment]
In any case, we should merge the DCO one.
@ajnavarro Maybe you can put the DCO in a separate PR, I think that should be merged ASAP.

**Pull request closed without merging**